### PR TITLE
Fix/update libs september

### PR DESCRIPTION
**Pull request by AndreiOla**
**Changed**
Updated plexus-utils lib, for fixing 2 MEDIUM vulnerabilities,

### CODE REVIEW

**Comment by AndreiOla** [comment]
Hey @tlawrie ,
Please add me to Contributors for this lib, so I can add Reviewers and push into the main branch.
Also please add yourself as reviewer for this PR, as I am not allowed to do so.
I need this approved, so I can tag and use the new lib version to other services.

**Comment by AndreiOla** [comment]
Hey @tlawrie  Please don't forget about this one.

File: pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 	<groupId>io.boomerang</groupId>
 	<artifactId>lib-eventing</artifactId>
-	<version>0.3.1</version>
+	<version>0.4.1</version>
```
Comment (corcoja) [inline pom.xml]: Increment version to `0.4.0` (after `0.4.0-rc.1`).

**Review by corcoja — CHANGES_REQUESTED** [review]

**Review by corcoja — APPROVED** [review]
LGTM! 👍🏻

**Review by tlawrie — APPROVED** [review]

**Pull request merged**